src/main/java/filter/expression/Qualifier.java

```diff
@@ -214,16 +192,16 @@ public static void setCurrentTime(LocalDateTime dateTime) {
     }
 
     public boolean isEmptyQualifier() {
-        return name.isEmpty() && content.isPresent() && content.get().isEmpty();
+        return type == QualifierType.EMPTY;
     }
 
     public boolean isFalseQualifier() {
-        return name.equalsIgnoreCase("false") && content.isPresent() && content.get().isEmpty();
+        return type == QualifierType.FALSE;
```

Comment:
If you decide to remove `Qualifier.EMPTY`, any solution to those methods returning `FilterExpression` or `Qualifier`?